flake.nix

```
@@ -155,7 +143,46 @@
           '';
         };
       };
-    in {
+    in rec {
+      packages = {
+        default = let 
+          package-lock = pkgs.purix.buildPackageLock {src = ./.;};
+          spago-lock = pkgs.purix.buildSpagoLock {src = ./.; corefn = true;};
+        in pkgs.stdenv.mkDerivation {
+          name = "registry-app";
+          version = "0.0.1";
+          src = ./app;
+          phases = ["buildPhase" "installPhase"];
+          nativeBuildInputs = [pkgs.purs pkgs.purs-backend-es pkgs.esbuild];
```

We don't actually need the compiler here, so we can take that out.

```suggestion
          nativeBuildInputs = [pkgs.purs-backend-es pkgs.esbuild];
```